docs/reference/configuration.md

```diff
@@ -146,16 +146,17 @@ No configuration options are supported by this scheduler type.
-| `dag_creation_poll_interval_secs` | Determines, in seconds, how often SQLMesh should check whether a DAG has been created (Default: `30`)                              |  int   |    N     |
-| `dag_creation_max_retry_attempts` | Determines the maximum number of attempts that SQLMesh will make while checking for whether a DAG has been created (Default: `10`) |  int   |    N     |
-| `backfill_concurrent_tasks`       | The number of concurrent tasks used for model backfilling during plan application (Default: `4`)                                   |  int   |    N     |
-| `ddl_concurrent_tasks`            | The number of concurrent tasks used for DDL operations like table/view creation, deletion, and so forth (Default: `4`)             |  int   |    N     |
+| Option                            | Description                                                                                                                         | Type   | Required |
+|-----------------------------------|-------------------------------------------------------------------------------------------------------------------------------------|:------:|:--------:|
+| `airflow_url`                     | The URL of the Airflow Webserver                                                                                                    | string | Y        |
+| `username`                        | The Airflow username                                                                                                                | string | Y        |
+| `password`                        | The Airflow password                                                                                                                | string | Y        |
+| `dag_run_poll_interval_secs`      | Determines, in seconds, how often a running DAG can be polled (Default: `10`)                                                       | int    | N        |
+| `dag_creation_poll_interval_secs` | Determines, in seconds, how often SQLMesh should check whether a DAG has been created (Default: `30`)                               | int    | N        |
+| `dag_creation_max_retry_attempts` | Determines the maximum number of attempts that SQLMesh will make while checking for whether a DAG has been created (Default: `10`)  | int    | N        |
+| `backfill_concurrent_tasks`       | The number of concurrent tasks used for model backfilling during plan application (Default: `4`)                                    | int    | N        |
+| `ddl_concurrent_tasks`            | The number of concurrent tasks used for DDL operations like table/view creation, deletion, and so forth (Default: `4`)              | int    | N        |
+| `max_snapshot_ids_per_request`    | The maximum number of snapshot IDs that can be sent in a single request to the Airflow Webserver in a GET request (Default: `None`) | int    | N        |
```

Comment:
why do we need this option? let's just limit it

Reply:
If a user maintains their own Airflow cluster they have more control over what the maximum request size is (unlike Cloud Composer). This is why I decided to make it configurable.